<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkActivityWidget</class>
 <widget class="QWidget" name="NetworkActivityWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>513</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="Tomahawk::Breadcrumb" name="breadCrumbLeft" native="true"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackLeft">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="PlaylistView" name="tracksViewLeft">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlaylistView</class>
   <extends>QTreeView</extends>
   <header>playlist/PlaylistView.h</header>
  </customwidget>
  <customwidget>
   <class>Tomahawk::Breadcrumb</class>
   <extends>QWidget</extends>
   <header>widgets/Breadcrumb.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
